src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/media/index.js

```diff
@@ -0,0 +1,65 @@
+import pick from 'ramda/src/pick';
+
+const generateVideoBlock = block => {
+  const generatedBlock = {
+    type: 'aresMediaMetadata',
+    blockId: `urn:bbc:ares::${block.subType}:${block.id}`,
+    model: pick(
+      [
+        'advertising',
+        'embedding',
+        'format',
+        'id',
+        'imageCopyright',
+        'imageUrl',
+        'subType',
+        'synopses',
+        'title',
+        'versions',
+      ],
+      block,
+    ),
+  };
+
+  if (generatedBlock.model.format === 'video') {
+    generatedBlock.model.format = 'audio_video';
+  }
+
+  return generatedBlock;
+};
+
+const generateImageBlock = block => {
```

Comment:
Not needed for this PR but in the future it might be nice to have all the Image block creation in `cpsAssetPage/convertToOptimoBlocks/blocks/image/index.js ` as Image will be used outside of media in the future

Reply:
Ah, cool, didn't realise the placeholder images use the same schema as regular images